src/SDKs/CognitiveServices/management/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.csproj

```diff
@@ -6,17 +6,21 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.CognitiveServices</PackageId>
 		<Description>Microsoft Azure Management Cognitive Services Library</Description>
-		<Version>2.0.0</Version>
+		<Version>2.1.0</Version>
 		<AssemblyName>Microsoft.Azure.Management.CognitiveServices</AssemblyName>
 		<PackageTags>Microsoft Azure Cognitive Services management;Cognitive Services;Cognitive Services management</PackageTags>
-		<PackageReleaseNotes/>
+		<PackageReleaseNotes />
 	</PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
 	</PropertyGroup>
 	<ItemGroup>
 		<Content Include="notice.txt" />
 	</ItemGroup>
+	
+	<ItemGroup>
```

Comment:
@felixwa please remove this item group as it's not needed to specify explicitly about directory structure.
Makes the project xml unnecessarily large.